pages/en/web/desktop-progressive-web-apps.md

```diff
@@ -20,15 +20,15 @@ resources:
 *[OTs]: Origin Trials
 *[OT]: Origin Trial
 
-Desktop [Progressive Web Apps](https://web.dev/what-are-pwas/) (PWAs) are a great cross-platform, cross-browser way to build apps with a security model that centers user safety and privacy and get built-in sharing with the web’s inherent linkability. Build new or enhance your existing web app with modern APIs to give them native-like capabilities, reliability, and installability. PWAs are the best way to deliver your web app for Chrome OS.
+Desktop [Progressive Web Apps](https://web.dev/what-are-pwas/) (PWAs) are a great cross-platform, cross-browser way to build apps with a security model that centers user safety and privacy and get built-in sharing with the web’s inherent linkability. Build new or enhance your existing web app with modern APIs to give them compiled app-like capabilities, reliability, and installability. PWAs are the best way to deliver your web app for Chrome OS.
```

Comment:
Not an expert, but don't you also compile PWAs? And Web Assembly apps?

Maybe "desktop app-like"?